python_binding/tox.ini

```diff
@@ -13,6 +13,6 @@ deps =
     pytest
     pytest-timeout
     aiounittest
-commands = pytest {posargs:tests} --timeout=300 -vvvvv
+commands = pytest {posargs:tests} --timeout=300 -vvvvv -s
```

Comment:
-s disables capturing, capturing simplifies debugging if the tests fail.

Reply:
I see. I found that without -s we couldn't see the log printed out to the stdout, so it's not easy to debug when the tests are stuck at some place. 
I can revert it back